src/NodeAnimator.tsx

```diff
@@ -72,7 +76,8 @@ export default class NodeAnimator extends React.Component<IProps> {
     private updateChildren(newChildren: React.ReactNode): void {
         const oldChildren = this.children || {};
         this.children = {};
-        React.Children.toArray(newChildren).forEach((c: ReactElement) => {
+        React.Children.toArray(newChildren).forEach((c) => {
+            if (!isReactElement(c)) return;
```

Comment:
Happy with this, just wondered if it may be a bit neater to chain a `.filter` using the typeguard as the predicate.

Reply:
Yes but it adds a whole 2nd iterator which feels a little overkill

Reply:
That is a fair point